### PR TITLE
Create python AST from string instead of file

### PR DESCRIPTION
**Pull request by swalsh15**
Use ast library instead of astor library to create python AST so we can use a string instead of file to build. Still works as expected and can print a "pretty" version of AST using astor.dump_tree().

### CODE REVIEW

File: translate/py_main.py [hunk 1]
```diff
@@ -13,11 +13,11 @@
...
-def py_to_gast(python_input_filename):
-    input_ast =  astor.code_to_ast.parse_file(python_input_filename)
+def py_to_gast(python_input):
+    input_ast = ast.parse(python_input)
     print(astor.dump_tree(input_ast))
     return py_router.node_to_gast(input_ast)
 
-fileName = "py_sample_code.py"
-gast = py_to_gast(fileName)
+file = "print('hello world')"
```
Comment (augerm) [inline translate/py_main.py:21]: Can we rename this to `src_code_input` (or something similar) now that this isn't a file anymore?

Comment (augerm) [reply translate/py_main.py:21]: `python_input` like in `py_to_gast` works

File: translate/py_main.py [hunk 2]
```diff
@@ -13,11 +13,11 @@
 takes pyton file and converts it to a node
```
[the remark this answers is not in the document]
Comment (jackdavidweber) [reply translate/py_main.py]: got it

File: translate/py_main.py (hunk 1 above)
Comment (jackdavidweber) [reply translate/py_main.py]: got it

**Pull request merged**